lib/stdlib/fields.juttle

```diff
@@ -0,0 +1,74 @@
+// Manipulate fields within a stream with these field reducers
+
+// Return an array of last N values of a field
+//
+// Parameters:
+//
+//   * field: ("value") field to tail
+//   * length: (1) number of recent values to retain
+export reducer tail(field, length=1) {
+    var data = [];
+
+    function update() {
+        if (*field != null) { 
```

Comment:
I'll wait for @dmajda @dmehra to vote on this one as I find it confusing if you have a mix of points and specifically state `values=tail(fieldx, 5)` but end up with values equal to [null, null, null] at the end of execution and the fieldx never never had any nulls itself. The other idea is to make it an option the reducer to `skipNulls`. The fact we can't distinguish between not there and set to null is a bit concerning.